### PR TITLE
Add kickstart announcement email

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Fixes https://github.com/srobo/tasks/issues/308

## To Do

- [x] Populate the final kickstart venues (namely Cambridge)

### CODE REVIEW

File: SR2020/2019-09-16-kickstart-announcement.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Kickstart is the event which starts the competition year. We invite all teams to attend the event, where we unveil the game, issue their kits, and help competitors get acquainted with the kit.
+
+## When / Where is kickstart?
+
+This year we're planning to host three Kickstart events, all on Saturday 26th October:
+
+- Cambridge (TBC)
+- The offices of [Thread](https://www.thread.com/)
+- The University of Southampton
```
Comment (trickeydan) [inline SR2020/2019-09-16-kickstart-announcement.md]:
```suggestion
- University of Southampton
```

Generally referred to with a "The".

File: SR2020/2019-09-16-kickstart-announcement.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## What is kickstart?
+
+Kickstart is the event which starts the competition year. We invite all teams to attend the event, where we unveil the game, issue their kits, and help competitors get acquainted with the kit.
+
+## When / Where is kickstart?
+
+This year we're planning to host three Kickstart events, all on Saturday 26th October:
+
+- Cambridge (TBC)
```
Comment (Adimote) [inline SR2020/2019-09-16-kickstart-announcement.md]: just adding a comment here to make sure you update this row before shipping :)

Comment (RealOrangeOne) [reply SR2020/2019-09-16-kickstart-announcement.md]: It's also in the PR description :+1:

**Comment by Adimote** [comment]
Scheduled to send at 08:30am in the morning!

**Pull request merged**